packages/adminrouter/extra/src/test-harness/tests/test_service.py

```diff
@@ -650,10 +650,10 @@ def test_if_ar_with_empty_cache_waits_for_mesos_during_service_resolve(
                                 headers=valid_user_header)
 
             t_spent = time.time() - t_start
-            assert resp.status_code == 200
-            data = resp.json()
-            data['endpoint_id'] == 'http://127.0.0.15:16001'
 
+        assert resp.status_code == 200
```

Comment:
I'd ask that at least there is a comment on the PR about the specifics. For me there is already too much mental state to keep track of in order to do a particularly useful review.